spec/controllers/sessions_controller_spec.rb

```diff
@@ -1,19 +1,31 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe Clearance::SessionsController do
   it { should be_a Clearance::BaseController }
 
-  describe 'on GET to /sessions/new' do
-    before { get :new }
+  describe "on GET to #new" do
+    context "when a user is not signed in" do
+      before { get :new }
 
-    it { should respond_with(:success) }
-    it { should render_template(:new) }
-    it { should_not set_the_flash }
+      it { should respond_with(:success) }
+      it { should render_template(:new) }
+      it { should_not set_the_flash }
+    end
+
+    context "when a user is signed in" do
+      before do
+        sign_in
+        get :new
+      end
+
+      it { should redirect_to(Clearance.configuration.redirect_url) }
```

Comment:
This is the line that made me think about the configuration values. When a user is signed in, it should not go to the `url_after_create` but the `url_for_signed_in_users`. They happen to be the same value in this case since the defaults are the same. But the expectation should read that it redirects to the "already signed in" URL.